test/test_manylinuxXXXX_only.py

```diff
@@ -80,6 +82,8 @@ def test(manylinux_image, tmp_path):
     expected_wheels = utils.expected_wheels(
         "spam", "0.1.0", manylinux_versions=platform_tag_map.get(manylinux_image, [manylinux_image])
     )
+    if manylinux_image in {"manylinux1"}:
+        expected_wheels = [w for w in expected_wheels if "-cp310-" not in w]
```

Comment:
Same as previous suggestion.

```suggestion
        expected_wheels = [w for w in expected_wheels if "-cp31" not in w]
```

Reply:
This one is a bit fiddlier, maybe mayeut's suggestion is fine, with a comment `# remove CPython 3.10 and above`